### ISSUE TITLE
Week 21:

### ISSUE THREAD

**Issue by SHuntzberry** [gw-sd-2016/ArtificialInstructor#7]
@twood02

This week I was able to get the application scoring in the right instances and the ability to write out to a file in order to have the results in a single location. Moving forward I now need to figure out a way to parse through the file created in order to produce some feedback for the users attempt. I will also be working on the UI aspect of showing the users the next notes. This section will have to be updated as the lesson increments notes and will show the next 5 notes in the lesson. By doing this I will be able to allow the user to know which notes are expected so they can focus more on playing and less on understanding the system. 

Side note for Professor Wood: Since we have not met since the Skype call before the semester started, would you like to meet this week? I think I may have a bit more to go over next week once I get the scoring properly adjusted and the new UI up and running, however, I would not mind meeting if you had any concerns as well.

**Comment by twood02** [comment]
yes, let's plan to meet this week so I can see your 70% demo and talk about your plans.  I"ll send out a schedule of times tomorrow.
